tests/test_simulate.py

```diff
@@ -358,8 +358,8 @@ def test_process_simulated_data():
 
 
 def test_retrieve_non_sparse_choices():
-    got = _retrieve_non_sparse_choices(
-        index=jnp.array([0, 3, 7]),
+    got = retrieve_non_sparse_choices(
+        indices=jnp.array([0, 3, 7]),
```

Comment:
In simulate.py, is there a reason for having `indices` and `index` differently in the public and private version of `retrieve_non_sparse_choices`? If not, this might be a good occasion to harmonise them.

Reply:
The reason was that the private function was written to work on a single index and then vmapped. Noticing that I only have to vmap `jnp.unravel_index` I rewrote this part for clarity.